app/client/reducers/metricReducer.js

```diff
@@ -0,0 +1,47 @@
+import {combineReducers} from "redux";
+import {DEEEEEEP} from "../utils";
+
+export const LOAD_METRICS_REQUEST = "LOAD_METRICS_REQUEST";
+export const LOAD_METRICS_SUCCESS = "LOAD_METRICS_SUCCESS";
+export const LOAD_METRICS_ERROR = "LOAD_METRICS_ERROR";
+export const UPDATE_METRICS = "UPDATE_METRICS";
+  
+const metricsState = (state = {}, action) => {
```

Comment:
make metrics a separate reducer. as we expand capabilities of dashboard this metrics reducer will be the place for all metric data requests/formatting